### PR TITLE
Refix locales

### PR DESCRIPTION
**Pull request by kay0u**
## The problem

...

## Solution

...

## PR Status

...

## How to test

...

## Validation

- [ ] Principle agreement 0/2 : 
- [ ] Quick review 0/1 : 
- [ ] Simple test 0/1 : 
- [ ] Deep review 0/1 :

### CODE REVIEW

**Comment by kay0u** [comment]
Should we use `diagnosis_mail_queue_too_big`?

File: data/hooks/diagnosis/24-mail.py [hunk 1]
```diff
@@ -54,16 +54,24 @@ def check_outgoing_port_25(self):
...
                            data={},
                            status="ERROR",
-                           summary="diagnosis_mail_ougoing_port_25_blocked",
-                           details=["diagnosis_mail_ougoing_port_25_blocked_details",
+                           summary="diagnosis_mail_outgoing_port_25_blocked",
+                           details=["diagnosis_mail_outgoing_port_25_blocked_details",
                                     "diagnosis_mail_outgoing_port_25_blocked_relay_vpn"])
+                problem_detected = True
+
+        if not problem_detected:
```
Comment (zamentur) [inline data/hooks/diagnosis/24-mail.py:70]: It's already done by the code in run() method.

**Comment by zamentur** [comment]
I have fixed the 2 chains (queue_too_big and ougoing in my branch)

**Comment by kay0u** [comment]
Thank you for your work! Let's close this PR!

**Pull request closed without merging**